.github/dependabot.yml

```
@@ -5,3 +5,10 @@ updates:
     schedule:
       interval: "daily"
     labels: ["dependencies"]
+  # Dependabot only updates hashicorp GHAs, external GHAs are managed by internal tooling (tsccr)
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-name: "hashicorp/*"
```

```suggestion
      - dependency-name: "hashicorp/*"
    groups:
      github-actions-breaking:
        update-types:
          - major
      github-actions-backward-compatible:
        update-types:
          - minor
          - patch
```

This is the config we use elsewhere in Core team repos which is nice because it allows you to merge the nonbreaking changes quickly while possibly taking more time to test breaking changes separately.